src/mudlet.h

```diff
@@ -384,6 +386,7 @@ class mudlet : public QMainWindow, public Ui::main_window
     QTextOption::Flags mEditorTextOptions;
 
     QPointer<QTableWidget> moduleTable;
+    QSystemTrayIcon mTrayIcon = QSystemTrayIcon(QIcon(":/icons/mudlet.png"));
```

Comment:
Maybe you can make the dev and ptb versions use different icons as they do already?

People on Linux will want to be able to customise the icon to be monochrome or something else, that would be nice sugar.

Reply:
At the moment I think I found quite elegant solution - tray icon will be taken directly from mudlet object.

Reply:
I agree, very nice.